modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -61,14 +65,7 @@ $ sudo restorecon -Rv rhcos_image_cache/
 $ export COMMIT_ID=$(/usr/local/bin/openshift-baremetal-install version | grep '^built from commit' | awk '{print $4}')
 ----
 +
-The ID determines which images the installer needs to download.
-
-. Get the URI for the {op-system} image that the installer will deploy on the nodes:
-+
-[source,terminal]
-----
-$ export RHCOS_OPENSTACK_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json  | jq .images.openstack.path | sed 's/"//g')
-----
+The ID determines which image the installer needs to download.
```

Comment:
```suggestion
The ID determines which image the installer must download.
```

> ! need to verb
Use "need to" only when "must" is too restrictive.

Reply:
Updated as suggested.